tracer/build/_build/Build.Steps.cs

```diff
@@ -320,14 +334,26 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
           }
           else
```

Comment:
nit: else if(IsLinux) to reduce nesting slightly

Reply:
done in [bc27402](https://github.com/DataDog/dd-trace-dotnet/pull/3447/commits/bc274021c24a19f561fb286e35a68530bb086679)